server.lua

```diff
@@ -11,11 +11,11 @@ lib.callback.register('qbx_scoreboard:server:getScoreboardData', function()
         if v then
             totalPlayers += 1
 
-            if v.PlayerData.job.type == "police" and v.PlayerData.job.onduty then
+            if v.PlayerData.job.type == 'police' and v.PlayerData.job.onduty then
                 policeCount += 1
             end
 
-            players[v.PlayerData.source].isOnDutyAdmin = IsPlayerAceAllowed(v.PlayerData.source, 'group.admin') and v.PlayerData.optin
+            players[v.PlayerData.source] = {isOnDutyAdmin = IsPlayerAceAllowed(source, 'admin') and v.PlayerData.optin}
```

Comment:
Where is source defined?

Reply:
IsPlayerAceAllowed(source, 'admin') was a mistake when submitted. Should be fixed in latest commit.